automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/SignInPage.scala

```diff
@@ -73,7 +73,6 @@ class GoogleSignInPopup(implicit webDriver: WebDriver) extends WebBrowser with W
     * Signs in to Google to authenticate for FireCloud.
     */
   def signIn(email: String, password: String): Unit = {
-    val chooseAccount = find(id("identifierLink")).filter(_.isDisplayed) foreach { click on _ }
```

Comment:
i'm assuming this turned out to be unnecessary, but it's definitely doing things